src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,64 @@
+package lotto.view;
+
+import lotto.domain.LottoTicket;
+import lotto.domain.LottoTickets;
+
+public class OutputView {
+
+    private OutputView() {
+    }
+
+    public static void printErrorMessage(String message) {
+        System.out.println(message);
```

Comment:
```suggestion
        System.err.println(message);
```

이런 것도 있답니다. 콘솔에서 어떻게 에러를 내뿜는지 확인해 보세요!

Reply:
코니의 피드백으로 System.err.println 에 대해 처음 알게 되어 사용해보았는데요, System.out.println으로 출력할 때에는 원하던 순서 (에러 메시지 출력 -> 재입력 받는 메시지 출력) 대로 출력이 잘 되었는데, err.println을 사용하니 (재입력 받는 메시지 출력 -> 에러 메시지 출력) 과 같이 순서가 변경되어 출력이 될 때도 있네요. 찾아보니 flush 타이밍 때문에 그런 것 같은데.. 이런 경우에는 어떻게 하는 것이 좋나요? 우선은 변경하지 않고 그대로 out.println으로 두었습니다!

Reply:
앗, 이거 저도 겪었던 문제인데 쓸 일이 없는 메서드여서 까먹었네요 😅